source/core/Synchronization.cpp

```diff
@@ -823,12 +823,12 @@ VIREO_FUNCTION_SIGNATURE4(QueueRef_Release, RefNumValType*, StringRef, TypedArra
         }
         else if (QueueRefNumManager::RefNumStorage().DisposeRefNum(refnumPtr->GetRefNum(), &queueRef) != kNIError_Success) {
             if (errPtr && !errPtr->status)
-                errPtr->SetError(true, kQueueArgErr, "ReleaseQueue");
+                errPtr->SetError(true, kQueueArgErr, "ReleaseQueue", true);
```

Comment:
Please remove all the 'true' at all the call-sites since the default is changed to 'true' now.